cdap-common/src/main/java/co/cask/cdap/common/logging/LocationAwareWrapperLogger.java

```diff
@@ -25,16 +26,32 @@
 /**
  * A {@link Logger} implementation that only emit logs that are accepted by {@link LogSampler}.
  */
-final class LocationAwareSamplingLogger implements Logger, LocationAwareLogger {
+final class LocationAwareWrapperLogger implements Logger, LocationAwareLogger {
 
-  private static final String FQCN = LocationAwareSamplingLogger.class.getName();
+  private static final String FQCN = LocationAwareWrapperLogger.class.getName();
 
   private final Logger logger;
   private final LogSampler sampler;
+  private final String mdcKey;
+  private final String mdcValue;
 
-  LocationAwareSamplingLogger(Logger logger, LogSampler sampler) {
+  LocationAwareWrapperLogger(Logger logger, LogSampler sampler) {
```

Comment:
Why change this class?? Isn't that we have a `LocationAwareMDCWrapperLogger` for that purpose??